test/e2e/bindata/assets/02_clusterrole.yaml

```diff
@@ -142,3 +142,23 @@ rules:
       - patch
       - update
       - watch
+  - apiGroups:
+      - kueue.x-k8s.io
+    resources:
+      - admissionchecks
```

Comment:
This can be removed also right?

Reply:
This isn't really hurting anything as its our test code so this can be a follow up.